va/tlsalpn.go

```diff
@@ -198,6 +199,30 @@ func checkExpectedSAN(cert *x509.Certificate, name identifier.ACMEIdentifier) er
 	return nil
 }
 
+// Confirm that of the OIDs provided, all of them are in the provided list of
+// extensions. Also confirms that of the extensions provided that none are
+// repeated. Per RFC8737, allows unexpected extensions.
+func checkAcceptableExtensions(exts []pkix.Extension, requiredOIDs []asn1.ObjectIdentifier) error {
+	oidSeen := make(map[string]bool)
+
+	for _, ext := range exts {
+		if oidSeen[ext.Id.String()] {
+			return fmt.Errorf("Extension OID %s seen twice", ext.Id)
+		}
+		oidSeen[ext.Id.String()] = true
+	}
+
+	for _, required := range requiredOIDs {
+		_, requiredOIDisPresent := oidSeen[required.String()]
```

Comment:
This is the nittiest of the nitpicks.

We use the value of oidSeen to compare on line:
https://github.com/letsencrypt/boulder/pull/5919/files#diff-4e6168e9f6abe1602cc5a74b18621918f115bd3a17066b293c103174955de386R209

And then here we use the `ok` return value when we get it from the map made above.

The default value for `bool` would be false here, so you could have it be like:

```suggestion
		requiredOIDisPresent , _ := oidSeen[required.String()]
```

Or just call it directly on line 218

Reply:
I'll remove `requiredOIDisPresent` as you suggest.